icu4c/source/common/unicode/urename.h

```diff
@@ -1295,7 +1303,6 @@
 #define uprv_convertToPosix U_ICU_ENTRY_POINT_RENAME(uprv_convertToPosix)
 #define uprv_copyAscii U_ICU_ENTRY_POINT_RENAME(uprv_copyAscii)
 #define uprv_copyEbcdic U_ICU_ENTRY_POINT_RENAME(uprv_copyEbcdic)
-#define uprv_currencyLeads U_ICU_ENTRY_POINT_RENAME(uprv_currencyLeads)
```

Comment:
It looks like this function still exists. (But it doesn't look like it is used though?)

Reply:
maybe dead stripped

Reply:
I notice a lot of variation in how the genren tool handles such internal functions, possible due to compiler variation or something. But yes, this does not seem to be used anywhere, it is internal with no U_EXPORT, etc. Can be dropped from urename.h